### PR TITLE
feat(statics): enable getting asset by id

### PR DESCRIPTION
**Pull request by CapnMigraine**
Ticket: BG-68438

### CODE REVIEW

File: modules/statics/src/map.ts [hunk 1]
```diff
@@ -44,7 +51,7 @@ export class CoinMap {
   }
 
   public has(key: string): boolean {
-    return this._map.has(key) || this._coinByAliases.has(key);
+    return this._map.has(key) || this._coinByIds.has(key) || this._coinByAliases.has(key);
```
Comment (sachushaji) [inline modules/statics/src/map.ts:54]: Can you kindly add a test case for this?

File: modules/statics/src/errors.ts [hunk 2]
```diff
@@ -26,6 +26,13 @@ export class DuplicateCoinDefinitionError extends BitGoStaticsError {
   }
 }
 
+export class DuplicateCoinIdDefinitionError extends BitGoStaticsError {
```
Comment (sachushaji) [inline modules/statics/src/errors.ts:29]: Can you kindly add a test case for this error flow?

File: modules/statics/src/map.ts (hunk 1 above)
Comment (sachushaji) [inline modules/statics/src/map.ts:54]: Can you kindly add a test case for this?

Comment (CapnMigraine) [reply modules/statics/src/map.ts:54]: done

File: modules/statics/src/errors.ts (hunk 2 above)
Comment (CapnMigraine) [reply modules/statics/src/errors.ts:29]: done

**Review by sachushaji — APPROVED** [review]

**Pull request merged**